Setup base of the app with CRA-v2,  CSS Modules and Sass

ka1130 opened the pull request:
This PR sets up the base of the application with [CRA-v2](https://github.com/facebook/create-react-app/issues/5103) that has CSS Modules and Sass already integrated. Thus, the app has not been yet ejected.

The default view has been changed to a sample header.

To keep the code clean, VSCode `prettier` plugin has been added with a basic configuration.

yakato commented on README.md:
```diff
@@ -1,2 +1,11 @@
+- `yarn install` to install dependencies
+- `yarn start` to run the server on `http://localhost:3000`
+
+Demo app to be find [here](https://ka1130.github.io/TVloVe/).
```
I believe that this shouldn't be here yet

yakato commented on src/components/App/Header/index.js:
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from './styles.module.scss';
```
You can change the name of scss file to name of the component. It will produce you better className for debugging.

ka1130 replied:
Wow, that's cool!

yakato commented on src/index.css:
```diff
@@ -0,0 +1,5 @@
+body {
```
Add come common resets here, please

ka1130 replied:
Sure, done. Hope I haven't added too much and it is fine not to add reset in a separate file in this case?

yakato replied:
It's enough to have it in this file. What about links `text-decoration`?

ka1130 commented:
Thanks, @yakato ! The trick with changing the naming convention of the style files is awesome, it's a shame I haven't used it before. Yes, I have copy-pasted the contents of `README.md` from another project, I've corrected it. I've also introduced the meyerweb CSS reset: please advice if this was a good move.

yakato commented:
Thanks for improvements. Perfect basic setup 👍
Re-consider resets as I commented and feel free to merge if it's ok for you.

ka1130 commented:
Oh, right, text-decoration should be reset too, thanks! :) Cool, I will add this and merge then. :)
